src/sagas/PersonalData.js

```diff
@@ -103,7 +103,18 @@ export function fetchAllPersonalData(config) {
 
 const getData = state => {
   const data = {
-    ...state.form.personal_data.values,
+    given_name: document
+      .getElementById("given_name")
+      .querySelector("input").value.trim(),
+    surname: document
+      .getElementById("surname")
+      .querySelector("input").value.trim(),
+    display_name: document
+      .getElementById("display_name")
+      .querySelector("input").value.trim(),
+    language: document
+      .getElementById("language")
+      .querySelector('input[type="radio"]:checked').value,
```

Comment:
Is it be possible to make this less dependent on the html? If not we should leave it, but I thought I'd ask as we might change the html in the future.

Reply:
yes! I have updated!